python/llm/example/CPU/QLoRA-FineTuning/alpaca-qlora/alpaca_qlora_finetuning_cpu.py

```diff
@@ -0,0 +1,330 @@
+#
```

Comment:
Can `alpaca_qlora_finetuning_cpu.py` and `qlora_finetuning_cpu.py` be combined? It seems they differ only in data-processing logic.
For example, maybe we can define a `datasetTokenizer` base class, use a unified map API in `qlora_finetuning_cpu.py`, and just reimplement the specific logic under the alpaca folder.
In this way, `qlora_finetuning_cpu.py` can also be ported to more datasets in the future.